game/scripts/vscripts/components/points/points.lua

```diff
@@ -9,62 +9,81 @@ end
 function PointsManager:Init ()
   DebugPrint ( 'Initializing.' )
 
-  PointsManager.hasGameEnded = false
+  self.hasGameEnded = false
 
-  CustomNetTables:SetTableValue( 'team_scores', 'score', {})
+  CustomNetTables:SetTableValue( 'team_scores', 'score', {
+    goodguys = 0,
+    badguys = 0,
+  })
```

Comment:
Custom Net Tables are weird. I don't want to mess with the any more than I need to.